### PR TITLE
Fixed a small bug related to multi-constrain k-way partitioning

### PR DESCRIPTION
**Pull request by ravil-mobile**
* added an option for a user to pass a vector of imbalances
* the current SeisSol's master branch references this branch

### CODE REVIEW

File: PartitionMetis.h [hunk 1]
```diff
@@ -56,16 +56,24 @@ class PartitionMetis
 #endif // USE_MPI
 
 #ifdef USE_MPI
+	enum Status {
```
Comment (krenzland) [inline PartitionMetis.h]: should be enum class

**Review by krenzland — CHANGES_REQUESTED** [review]
Looks mostly good, just one tiny change.

**Review by Thomas-Ulrich — APPROVED** [review]

**Comment by krenzland** [comment]
@ravil-mobile Is this PR still relevant?

**Comment by Thomas-Ulrich** [comment]
I think it is (see other PR, extending this PR, required for Edgeweigts constraints).

File: PartitionMetis.h (hunk 1 above)
Comment (ravil-mobile) [reply PartitionMetis.h]: done

**Review by krenzland — APPROVED** [review]

**Pull request merged**